pandas-stubs/core/series.pyi

```diff
@@ -179,7 +179,13 @@ class _LocIndexerSeries(_LocIndexer, Generic[S1]):
     @overload
     def __getitem__(
         self,
-        idx: MaskType | Index | Sequence[float] | list[str] | slice | _IndexSliceTuple,
+        idx: MaskType
+        | Index
+        | Sequence[float]
+        | list[str]
+        | slice
+        | _IndexSliceTuple
+        | Callable,
```

Comment:
Happy to keep it as is - it is probably not worth to try to enumerate all allowed arguments/return values.

Reply:
Yes.  We have it this way in `DataFrame.loc` as well.